samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java

```diff
@@ -489,6 +489,7 @@ public void sendChatMessage(View view) {
                 logAndToast("Message sent with ID: " + chatMessageId);
             } catch (InterruptedException | ExecutionException e) {
                 logAndToast("Send message failed: " + e.getMessage());
+                logAndToast("Send message failed: " + e.getStackTrace());
```

Comment:
We shouldn't repeat the same message at the beginning, people will think sending the message failed twice. Furthermore, toasting the stacktrace is not user friendly at all, we should just log it in my opinion.

Reply:
Removed this line. The toast operation shows a message on the application to notify a user something is wrong. This makes sense for a demo application from my opiontion

Reply:
I understand the intent, I just think the stacktrace message might not be even readable in a toast. If you think it is, feel free to leave it in.